plugins/woocommerce/src/Admin/Marketing/InstalledExtensions.php

```diff
@@ -319,6 +377,200 @@ protected static function get_mailpoet_extension_data() {
+		$data['icon'] = WC_ADMIN_IMAGES_FOLDER_URL . '/marketing/tiktok.jpg';
+
+		if ( 'activated' === $data['status'] ) {
+			if ( false !== get_option( 'tt4b_access_token' ) ) {
+				$data['status'] = 'configured';
+			}
+
+			$data['settingsUrl'] = admin_url( 'admin.php?page=tiktok' );
+			$data['docsUrl']     = 'https://woocommerce.com/document/tiktok-for-woocommerce/';
+			$data['supportUrl']  = 'https://ads.tiktok.com/athena/user-feedback/?identify_key=6a1e079024806640c5e1e695d13db80949525168a052299b4970f9c99cb5ac78';
+		}
+
+		return $data;
+	}
+
+	/**
+	 * Get Jetpack CRM for WooCommerce extension data.
+	 *
+	 * @return array|bool
+	 */
+	protected static function get_jetpack_crm_extension_data() {
+		$slug = 'zero-bs-crm';
+
+		if ( ! PluginsHelper::is_plugin_installed( $slug ) ) {
+			return false;
+		}
+
+		$data         = self::get_extension_base_data( $slug );
+		$data['icon'] = WC_ADMIN_IMAGES_FOLDER_URL . '/marketing/jetpack-crm.png';
+
+		if ( 'activated' === $data['status'] ) {
+			$data['status']      = 'configured';
+			$data['settingsUrl'] = admin_url( 'admin.php?page=zerobscrm-plugin-settings' );
+			$data['docsUrl']     = 'https://kb.jetpackcrm.com/';
+			$data['supportUrl']  = 'https://kb.jetpackcrm.com/crm-support/';
+		}
+
+		return $data;
+	}
+
+	/**
+	 * Get WooCommerce Zapier extension data.
+	 *
+	 * @return array|bool
+	 */
+	protected static function get_zapier_extension_data() {
+		$slug = 'woocommerce-zapier';
+
+		if ( ! PluginsHelper::is_plugin_installed( $slug ) ) {
+			return false;
+		}
+
+		$data         = self::get_extension_base_data( $slug );
+		$data['icon'] = WC_ADMIN_IMAGES_FOLDER_URL . '/marketing/zapier.png';
+
+		if ( 'activated' === $data['status'] ) {
+			$data['status']      = 'configured';
+			$data['settingsUrl'] = admin_url( 'admin.php?page=wc-settings&tab=wc_zapier' );
+			$data['docsUrl']     = 'https://docs.om4.io/woocommerce-zapier/';
+		}
+
+		return $data;
+	}
+
+	/**
+	 * Get Salesforce extension data.
+	 *
+	 * @return array|bool
+	 */
+	protected static function get_salesforce_extension_data() {
+		$slug = 'integration-with-salesforce';
+
+		if ( ! PluginsHelper::is_plugin_installed( $slug ) ) {
+			return false;
+		}
+
+		$data         = self::get_extension_base_data( $slug );
+		$data['icon'] = WC_ADMIN_IMAGES_FOLDER_URL . '/marketing/salesforce.jpg';
+
+
+		if ( 'activated' === $data['status'] && class_exists( '\Integration_With_Salesforce_Admin' ) ) {
+			if ( ! method_exists( '\Integration_With_Salesforce_Admin', 'get_connection_status' ) || \Integration_With_Salesforce_Admin::get_connection_status() ) {
+				$data['status'] = 'configured';
+			}
+
+			$data['settingsUrl'] = admin_url( 'admin.php?page=integration-with-salesforce' );
+			$data['docsUrl']     = 'https://woocommerce.com/document/salesforce-integration/';
+			$data['supportUrl']  = 'https://wpswings.com/submit-query/?utm_source=wpswings-salesforce-woo&utm_medium=woo-backend&utm_campaign=submit-query';
```

Comment:
Is the query string needed?

 `?utm_source=wpswings-salesforce-woo&utm_medium=woo-backend&utm_campaign=submit-query`